index.mako

```diff
@@ -1579,6 +1579,17 @@ cache:
   - $HOME/.cache/pre-commit
 ```
 
+### gitlab example
+
+```yaml
+variables:
+  XDG_CACHE_HOME: "$CI_PROJECT_DIR/.cache/"
```

Comment:
I don't think this is correct, this puts the pre-commit cache inside the clone of the repository (which is likely to have negative side-effects)

Reply:
OK. I'm actively investigating this right now.

Reply:
Just following on from a conversation you've had earlier. Will see what effect this has on our repo.
https://github.com/pre-commit/pre-commit/issues/1207